### DISCUSSION TITLE
Is it normal to have the container utils by default?

### DISCUSSION THREAD

**Discussion by babakfp** [tailwindlabs/tailwindcss#9607] [Help] [answered]
Hi

![Screenshot 2022-10-19 082355](https://user-images.githubusercontent.com/44144724/196735221-bc7dde02-d977-4a84-a20a-9bd729f109f3.png)

As far as I know, you need to add some stuff to the config file to get the utils for the container, but I haven't done any of that.

Thanks

**Comment by wongjn** [comment, marked as answer]
It *is* normal to have the container **component** (it is not a utility) by default. You can read more about the [`container` component in the documentation](https://tailwindcss.com/docs/container)

**Reply by babakfp** [reply]
Thanks
Can I completely remove it? Because I'm using a custom utility named `.container` and `.container-fluid`, so...

**Reply by wongjn** [reply]
Yes you can, as per the [documentation on core plugins configuration](https://tailwindcss.com/docs/configuration#core-plugins), you could do something like this:
```js
module.exports = {
  // …
  corePlugins: {
    container: false,
  },
  // …
};
```
